dao/src/main/java/com/epam/esm/entity/GiftCertificate.java

```diff
@@ -0,0 +1,128 @@
+package com.epam.esm.entity;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+import java.util.StringJoiner;
+
+/**
+ * GiftCertificate entity.
+ *
+ * @since 1.0
+ */
+public class GiftCertificate {
+
+    private Long id;
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private int duration;
+    private String createDate;
+    private String lastUpdateDate;
+
+
+    public GiftCertificate() {
+    }
+
+    public GiftCertificate(Long id, String name, String description, BigDecimal price, int duration, String createDate, String lastUpdateDate) {
```

Comment:
If you really need such big constructor, I think builder will be better

Reply:
I removed all args constructors from GiftCertificate classes.

commit 301869b